### PR TITLE
Replace read inline

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: src/relay/src/__flowtests__/readInlineData.js [hunk 1]
```diff
@@ -1,94 +0,0 @@
-// @flow
```
Comment (mrtnzlml) [inline src/relay/src/__flowtests__/readInlineData.js:1]: Why is this test not valid anymore? It should still work, no?

Comment (tbergq) [reply src/relay/src/__flowtests__/readInlineData.js:1]: Do we really need it though? We are just exporting the type from `react-relay`

Comment (mrtnzlml) [reply src/relay/src/__flowtests__/readInlineData.js:1]: It's there to be sure it still works as expected. How do we know the types still work now?

Comment (tbergq) [reply src/relay/src/__flowtests__/readInlineData.js:1]: If they don't work, how will we fix them?

Comment (mrtnzlml) [reply src/relay/src/__flowtests__/readInlineData.js:1]: We add the `readInlineData` wrapper back just like we did for years. 🙂

**Pull request merged**